[IMP] Maintain: Add process to delete account and clean-up (#3443)
odoo/documentation: content/applications/general/users/delete_account.rst

===========================

> Odoo.com Account Management

Reviewer: Wondering—
If this doc is exclusively about deleting an Odoo account, then wouldn't we want to name it as such? Or are we planning to add more content to this doc later?

Author: I've renamed the doc: Deleting Odoo.com Account. Great suggestion.

===========================

Account management
==================

Overview
--------

To delete your Odoo account, access the delete option by clicking on the user icon. The delete
option can be accessed by going to :menuselection:`My Account --> Edit Security Settings -->
Delete Account`. It can also be accessed by going to https://www.odoo.com/my/home.

Deletion Process
----------------

> Click on the user icon and select :guilabel:`My Account`. Then click on :guilabel:`Edit Security
> Settings`.
> 
> Next, under the :guilabel:`Delete Account` section, click on the :guilabel:`Delete Account` button.

Reviewer: Assuming this doc is *only* about account deletion (and we're not planning to revise the overview later to make room for new content), then either the Overview or the first sentence should be revised since it's redundant.

Suggestion below:

```suggestion
On the :guilabel:`Security Settings` page, scroll to the :guilabel:`Delete Account` section, and
then click on the :guilabel:`Delete Account` button.
```

Author: Noted, I've reduced the number of sentences. Thanks.

.. danger::

>    Deleting an Odoo account is irreversible. Take caution when performing this action as the Odoo.com
>    account will not be retrievable.

Reviewer: Break earlier on line 22
```suggestion
   Deleting an Odoo account is irreversible. Take caution when performing this action as the
   Odoo.com account will not be retrievable.
```

> Upon clicking the :guilabel:`Delete Account` button Odoo will populate a pop-up window verifying the
> change to delete the account.

Reviewer: Sentence reads weird imo.

Wording suggestion to simplify it:
```suggestion
Upon clicking the :guilabel:`Delete Account` button, a pop-up window will appear, asking you to
confirm the account deletion.
```

Author: 👍 All changed! Thanks.
